cli/cli.ts

```diff
@@ -1543,30 +1562,36 @@ function updateDefaultProjects(cfg: pxt.TargetBundle) {
             files: {}
         };
 
-        if (!fs.existsSync(projectPath)) {
+        if (!fs.existsSync(templatePath)) {
```

Comment:
isn't templatePath guaranteed to exist at this point? getTemplates() returns existing paths.